commit 4d0c57234cac6512feda000e9974596ce037f45b
Author: Peter Rajnoha <prajnoha@redhat.com>
Date:   Wed Jun 29 15:58:14 2016 +0200

    libdm: do not issue 'Failed to create directory' message for failure in dm_create_dir
    
    There are detailed messages inside _create_dir_recursive that
    dm_create_dir calls (except EROFS which where the message is not
    generated, like anywhere else in the code).
---
 libdm/libdm-file.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/libdm/libdm-file.c b/libdm/libdm-file.c
index 9867824..2c313b3 100644
--- a/libdm/libdm-file.c
+++ b/libdm/libdm-file.c
@@ -97,10 +97,8 @@ int dm_create_dir(const char *dir)
 	if (stat(dir, &info) == 0 && S_ISDIR(info.st_mode))
 		return 1;
 
-	if (!_create_dir_recursive(dir)) {
-		log_error("Failed to create directory %s.", dir);
-		return 0;
-	}
+	if (!_create_dir_recursive(dir))
+		return_0;
 
 	return 1;
 }
